data/human/human missions.txt

```diff
@@ -5416,3 +5416,345 @@ mission "Deal Change 2"
 				die
 			label skip
 			`	After a few minutes, the pirates finish unloading the cargo, and Ci steps forward, holding out a credit chip. "<payment>, for making the right choice." Ci and the pirates return to their ships as you take off and head towards the main spaceport of <planet>.`
+
+
+
+mission "Karen Transport 1"
+	name "Passenger transport to <planet>"
+	description "Bring <passengers> to <destination>."
+	passengers 1
+	deadline 2 0
+	source
+		government "Republic"
+		attributes "paradise"
+	destination
+		distance 1 1
+		attributes "paradise"
+	to offer
+		random < 1
+		"flagship crew" > 3
+	on offer
+		conversation
+			`As you are wandering the spaceport, a woman with extravagant clothing approaches you. "I am to be transported to <destination> immediately. You will be paid well."`
+			choice
+				`	(Invite her into your ship.)`
+					goto accept
+				`	"I'd expect my passengers to be more respectful."`
+			`	"Well, you just lost yourself a passenger!" She storms off angrily, cursing the audacity of "these space freaks."`
+				decline
+			
+			label accept
+			`	She takes off towards your ship without waiting for you. When you catch up to her, she is standing in a passageway, shouting at your crew to take her bags away. You tell her that your crew is busy and passengers are expected to carry their own luggage then leave for your cabin.`
+				accept
+	on enter
+		dialog
+			`During takeoff, you keep hearing loud banging and cursing from one of the passenger cabins. Later on, you hear the woman you just picked up screeching at your crew for their 'gross incompetence' and 'boneheaded flying.'`
+			`	You can barely restrain yourself from making a snappy remark, but you have learned that it's best not to antagonize your passengers.`
+	on enter
+		conversation
+			`You hear hurling and splashing noises in the background. It seems your passenger threw up, not uncommon for first-timers.`
```

Comment:
One caveat here is that a player can trigger this dialog by just taking off and landing again. Probably not worth dropping it for that edge case, but in case we ever get something like, `on enter destination`, that would be worth swapping it over.